### PR TITLE
feat: Update footer with new design

### LINKED ISSUE #28: Update footer with new design
Screenshot 2021-01-25 at 18 02 57

### PR DESCRIPTION
**Pull request by TanyaDolgopolova**
Close #28

### CODE REVIEW

File: web/src/gatsby-plugin-theme-ui/index.ts [hunk 1]
```diff
@@ -87,6 +87,11 @@ const theme = {
       textAlign: 'left',
       borderBottomStyle: 'solid',
     },
+    a: {
+      color: 'background',
+      textDecoration: 'underline',
+      fontSize: 3
```
Comment (cobraz) [inline web/src/gatsby-plugin-theme-ui/index.ts:93]: Should probably not define `fontSize` on `a`.

File: web/src/layouts/components/Footer.tsx [hunk 2]
```diff
@@ -1,43 +1,109 @@
 import React from 'react'
-import { Box, Text } from 'theme-ui'
+import { Box, Button, Flex, Link, Text } from 'theme-ui'
 import Container from '../../components/Container'
 import Logo from './Logo'
-import Link from '../../components/Link'
-import ALink from '../../components/ALink'
+import SocialGithubSvg from '../../assets/Social-Github.svg'
+import SocialLinkedinSvg from '../../assets/Social-Linkedin.svg'
+import SocialFacebookSvg from '../../assets/Social-Facebook.svg'
```
Comment (cobraz) [inline web/src/layouts/components/Footer.tsx:7]: SVGs should be inside in a React component. An example can be found here:

https://github.com/tabetalt/kit/blob/master/src/icons/ArrowsIcon.tsx

File: web/src/layouts/components/Footer.tsx [hunk 3]
```diff
@@ -1,43 +1,109 @@
...
+              rel="noopener noreferrer"
+              sx={{
+                width: '21px', 
+                ml: 2, 
+                cursor: 'pointer'
+              }}
+            >
+              <SocialFacebookSvg />
+            </Link>
+          </Flex>
```
Comment (cobraz) [inline web/src/layouts/components/Footer.tsx:106]:
```suggestion
        <Flex sx={{justifyContent: 'flex-end'}}>
        	{[
				{ href: 'https://github.com/bjerkio', icon: <SocialGithubSvg /> },
				{ href: 'https://www.linkedin.com/company/bjerk/', icon: <SocialLinkedinSvg /> },
				{ href: 'https://www.facebook.com/WeAreDigitalization', icon: <SocialFacebookSvg /> },
			].map({href, icon} => (
            <Link 
              href={href}
              target="_blank"
              rel="noopener noreferrer"
              sx={{
                width: '21px', 
                ml: 2, 
                cursor: 'pointer'
              }}
            >
              {icon}
            </Link>))}
          </Flex>
```

**Review by cobraz — APPROVED** [review]

**Comment by TanyaDolgopolova** [comment]
@cobraz what actions need to be after clicking on phone number or email?

**Comment by cobraz** [comment]
> @cobraz what actions need to be after clicking on phone number or email?

Probably enough with `tel:` for phone number and email no link (for now)?

**Comment by TanyaDolgopolova** [comment]
> > @cobraz what actions need to be after clicking on phone number or email?
> 
> Probably enough with `tel:` for phone number and email no link (for now)?

Okay 👌

File: web/src/layouts/components/Footer.tsx (hunk 2 above)
Comment (TanyaDolgopolova) [reply web/src/layouts/components/Footer.tsx:7]: As I know this SVG icons are created via js script. Maybe we need the same?

Comment (cobraz) [reply web/src/layouts/components/Footer.tsx:7]: You can copy the script, if you'd like :)

Comment (TanyaDolgopolova) [reply web/src/layouts/components/Footer.tsx:7]: Yeah, but at the last time that script not worked. Did you fix that?

**Comment by AnnaBjerk** [comment]
If this is approved, can we merge and close? @TanyaDolgopolova

**Pull request merged**